src/index.ts

```diff
@@ -1733,6 +1734,9 @@ function createStore<S>(initial: SetInitialStateAction<S>): Store {
     return new Store(initialValue);
 }
 
+// Do not try to use useLayoutEffect if DOM not available (SSR)
```

Comment:
1. place url link here in comments to the ticket you are addressing
2. to fix the build update tsconfig.prod.json, add 'dom' library: `"lib": [ "dom" ],`

Reply:
went for sleep updating in few minutes